message/validation/consensus_state.go

```diff
@@ -0,0 +1,34 @@
+package validation
+
+import (
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	"github.com/cornelk/hashmap"
+)
+
+// ConsensusID uniquely identifies a public key and role pair to keep track of state.
+type ConsensusID struct {
+	PubKey phase0.BLSPubKey
+	Role   spectypes.BeaconRole
+}
+
+// ConsensusState keeps track of the signers for a given public key and role.
+type ConsensusState struct {
+	// TODO: consider evicting old data to avoid excessive memory consumption
```

Comment:
we need to create a task about this. 
@Tomblox1  - Evict hashmap records of Signers in `validation/conesnsus_state.go`

Reply:
@Tomblox1 lets resolve when this task is open.